acquisitions/src/main/resources/domain/mod-orders/features/should-decrease-quantity-when-delete-piece-with-no-location.feature

```diff
@@ -0,0 +1,211 @@
+@parallel=false
+Feature: Should decrease quantity when delete piece with no location
```

Comment:
Why are there changes in mod-orders tests?

Reply:
This occurs when I rebase with masters